mk/main.mk

```diff
@@ -143,6 +143,11 @@ ifdef CFG_ENABLE_DEBUGINFO
   CFG_RUSTC_FLAGS += -g
 endif
 
+ifdef CFG_ENABLE_DEBUGINFO_LINES
+  $(info cfg: enabling line number debuginfo (CFG_ENABLE_DEBUGINFO_LINES))
+  CFG_RUSTC_FLAGS += -C debuginfo=1
```

Comment:
If I use `./configure --enable-debuginfo --release-channel=stable`, will I still get full debuginfo?  It looks like the implicit debuginfo-lines will override that choice here.